--- origsrc/SDL_mixer-1.2.11/configure.in	2009-11-12 00:41:28.000000000 -0600
+++ src/SDL_mixer-1.2.11/configure.in	2009-11-17 02:08:58.236937100 -0600
@@ -148,6 +148,7 @@ dnl Function to find a library in the co
 find_lib()
 {
     gcc_bin_path=[`$CC -print-search-dirs 2>/dev/null | fgrep programs: | sed 's/[^=]*=\(.*\)/\1/' | sed 's/:/ /g'`]
+    env_bin_path=[`echo $PATH | sed 's/:/ /g'`]
     gcc_lib_path=[`$CC -print-search-dirs 2>/dev/null | fgrep libraries: | sed 's/[^=]*=\(.*\)/\1/' | sed 's/:/ /g'`]
     env_lib_path=[`echo $LIBS $LDFLAGS $* | sed 's/-L[ ]*//g'`]
     if test "$cross_compiling" = yes; then
@@ -155,7 +156,7 @@ find_lib()
     else
         host_lib_path="/usr/$base_libdir /usr/local/$base_libdir"
     fi
-    for path in $gcc_bin_path $gcc_lib_path $env_lib_path $host_lib_path; do
+    for path in $gcc_bin_path $env_bin_path $gcc_lib_path $env_lib_path $host_lib_path; do
         lib=[`ls -- $path/$1 2>/dev/null | sort | sed 's/.*\/\(.*\)/\1/; q'`]
         if test x$lib != x; then
             echo $lib
@@ -252,7 +253,10 @@ return 1;
             *-*-darwin*)
                 mikmod_lib=[`find_lib libmikmod.dylib`]
                 ;;
-            *-*-cygwin* | *-*-mingw32*)
+            *-*-cygwin*)
+                mikmod_lib=[`find_lib "cygmikmod-*.dll"`]
+                ;;
+            *-*-mingw32*)
                 mikmod_lib=[`find_lib "mikmod*.dll"`]
                 ;;
             *)
@@ -349,7 +353,10 @@ if test x$enable_music_ogg = xyes; then
                         ogg_lib=[`find_lib libvorbisidec.[0-9]*`]
                     fi
                     ;;
-                *-*-cygwin* | *-*-mingw32*)
+                *-*-cygwin*)
+                    ogg_lib=[`find_lib "cygvorbisidec-*.dll"`]
+                    ;;
+                *-*-mingw32*)
                     ogg_lib=[`find_lib "vorbisidec*.dll"`]
                     ;;
                 *)
@@ -378,7 +385,10 @@ if test x$enable_music_ogg = xyes; then
                 *-*-darwin*)
                     ogg_lib=[`find_lib libvorbisfile.dylib`]
                     ;;
-                *-*-cygwin* | *-*-mingw32*)
+                *-*-cygwin*)
+                    ogg_lib=[`find_lib "cygvorbisfile-*.dll"`]
+                    ;;
+                *-*-mingw32*)
                     ogg_lib=[`find_lib "libvorbisfile*.dll"`]
                     ;;
                 *)
@@ -444,7 +454,10 @@ int main( int argc, char *argv[] ) {
                 *-*-darwin*)
                     flac_lib=[`find_lib libFLAC.dylib`]
                     ;;
-                *-*-cygwin* | *-*-mingw32*)
+                *-*-cygwin*)
+                    flac_lib=[`find_lib "cygFLAC-*.dll"`]
+                    ;;
+                *-*-mingw32*)
                     flac_lib=[`find_lib "libFLAC*.dll"`]
                     ;;
                 *)
@@ -479,7 +492,10 @@ if test x$enable_music_mp3 = xyes; then
             *-*-darwin*)
                 smpeg_lib=[`find_lib libsmpeg.dylib`]
                 ;;
-            *-*-cygwin* | *-*-mingw32*)
+            *-*-cygwin*)
+                smpeg_lib=[`find_lib "cygsmpeg-*.dll"`]
+                ;;
+            *-*-mingw32*)
                 smpeg_lib=[`find_lib "smpeg*.dll"`]
                 ;;
             *)
