--- origsrc/SDL_mixer-1.2.8/configure.in	2007-07-21 02:14:57.000000000 -0500
+++ src/SDL_mixer-1.2.8/configure.in	2007-07-23 18:47:47.937500000 -0500
@@ -37,16 +37,11 @@
 AC_SUBST(LT_AGE)
 
 dnl Detect the canonical build and host environments
-AC_CONFIG_AUX_DIRS($srcdir/build-scripts)
+AC_CONFIG_AUX_DIR(build-scripts)
 AC_CANONICAL_HOST
 
 dnl Set up the compiler and linker flags
 case "$host" in
-    *-*-cygwin*)
-        # We build SDL on cygwin without the UNIX emulation layer
-        BASE_CFLAGS="-I/usr/include/mingw -mno-cygwin"
-        BASE_LDFLAGS="-mno-cygwin"
-        ;;
     *)
         BASE_CFLAGS="-D_GNU_SOURCE=1"
         BASE_LDFLAGS=""
@@ -86,7 +81,7 @@
 AC_C_CONST
 
 case "$host" in
-    *-*-cygwin* | *-*-mingw32*)
+    *-*-mingw32*)
         SOURCES="$srcdir/version.rc"
         EXE=".exe"
         if test "$build" != "$host"; then # cross-compiling
