--- origsrc/SDL_mixer-1.2.8/configure.in	2008-12-23 14:36:29.006125000 -0600
+++ src/SDL_mixer-1.2.8/configure.in	2008-12-23 14:42:57.412375000 -0600
@@ -115,7 +115,7 @@
     gcc_bin_path=[`$CC -print-search-dirs 2>/dev/null | fgrep programs: | sed 's/[^=]*=\(.*\)/\1/' | sed 's/:/ /g'`]
     gcc_lib_path=[`$CC -print-search-dirs 2>/dev/null | fgrep libraries: | sed 's/[^=]*=\(.*\)/\1/' | sed 's/:/ /g'`]
     env_lib_path=[`echo $LIBS $LDFLAGS | sed 's/-L[ ]*//g'`]
-    for path in $gcc_bin_path $gcc_lib_path $env_lib_path /usr/lib /usr/local/lib; do
+    for path in $gcc_bin_path /usr/bin $gcc_lib_path $env_lib_path /usr/lib /usr/local/lib; do
         lib=[`ls -- $path/$1 2>/dev/null | sort -r | sed 's/.*\/\(.*\)/\1/; q'`]
         if test x$lib != x; then
             echo $lib
@@ -292,7 +292,10 @@
                 *-*-darwin*)
                     ogg_lib=[`find_lib libvorbisfile*.dylib`]
                     ;;
-                *-*-cygwin* | *-*-mingw32*)
+                *-*-cygwin*)
+                    ogg_lib=[`find_lib "cygvorbisfile-*.dll"`]
+                    ;;
+                *-*-mingw32*)
                     ogg_lib=[`find_lib "libvorbisfile*.dll"`]
                     ;;
                 *)
@@ -327,7 +330,10 @@
             *-*-darwin*)
                 smpeg_lib=[`find_lib libsmpeg*.dylib`]
                 ;;
-            *-*-cygwin* | *-*-mingw32*)
+            *-*-cygwin*)
+                smpeg_lib=[`find_lib "cygsmpeg-*.dll"`]
+                ;;
+            *-*-mingw32*)
                 smpeg_lib=[`find_lib "smpeg*.dll"`]
                 ;;
             *)
